### DISCUSSION TITLE
Hotkeys for Volume and add DDC/CI commends like change monitor input device

### DISCUSSION THREAD

**Discussion by poorya96** [xanderfrangos/twinkle-tray#613] [Ideas]
Thanks for the very useful App.
I have two suggestions
1. Adding Hotkeys to the volume control can be very useful.
2. also, adding DDC / CI commands, such as changing the monitor input device, can be very useful.
Thanks

**Comment by xanderfrangos** [comment]
Hi @P0orya, thanks for the suggestion! I get this one a lot. I plan to eventually add support for additional DDC/CI command hotkeys, but I don't know when at this time.
